### ISSUE TITLE
ModuleNotFoundError on idlelib

### ISSUE THREAD

**Issue by mimminou** [Nuitka/Nuitka#1564]
Problem :
After compiling, trying to run Nuitka will output this error :

from idlelib.tooltip import Hovertip
ModuleNotFoundError: No module named 'idlelib'

What I'm trying to do is import Hovertips for usage with Tk-Inter. Everything works normally if i remove this import and all the stuff that depends on it.

python -m nuitka --version
Windows : 

> 0.7.7
> Commercial: None
> Python: 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:34:34) [MSC v.1928 32 bit (Intel)]
> Flavor: Unknown
> Executable: C:\Users\bL4nK\AppData\Local\Programs\Python\Python38-32\python.exe
> OS: Windows
> Arch: x86
> WindowsRelease: 10

Linux :

> 0.7.7
> Commercial: None
> Python: 3.10.4 (main, Mar 25 2022, 00:00:00) [GCC 12.0.1 20220308 (Red Hat 12.0.1-0)]
> Flavor: Unknown
> Executable: /home/x1/Desktop/pdb2pqr-contrib/bin/python
> OS: Linux
> Arch: x86_64
> Distribution: Fedora 36

* How did you install Nuitka and Python

through pip, both globally and in virtualenv.

* The specific PyPI names and versions

idlelib isn't installed through pypi, it is similar to Tkinter, rather it comes preinstalled with python ( on Windows ) or through a package manager (rpm).

* Many times when you get an error from Nuitka, your setup may be special
Nuitka command on Windows :

nuitka pdbaser.py --plugin-enable=tk-inter --plugin-enable=numpy --standalone --windows-disable-console --windows-icon-from-ico=default_icon.ico --include-data-file=default_icon.ico=default_icon.ico --include-data-file=Main.ui=Main.ui --include-data-file=vc_redist.x86_2015.exe=vc_redist.x86_2015.exe --include-data-file=LiberationSans-Regular.ttf=LiberationSans-Regular.ttf --include-data-file=OpenSans-SemiBold.ttf=OpenSans-SemiBold.ttf --windows-force-stderr-spec=errors.txt --windows-force-stdout=Slog.txt --msvc=14.0 --enable-plugin=anti-bloat --noinclude-pytest-mode=nofollow --noinclude-setuptools-mode=nofollow

Example :

Run anything while importing anything from idlelib
from idlelib.tooltip import Hovertip

**Comment by kayhayen** [comment]
The `idlelib` was excluded on purpose once upon a time ago, it is large. But we now got a better way, that is non-automatic standard library inclusion, which was new in 0.7, which should address this. I added it to factory, so the bug should be gone.

Basically these are standard library modules that are not included outright, but only if found used. Things not there, are included, and not tracked. And things excluded, which `idlelib` was, are apparently not includable, which is bad. With the new tool, we can just make it non-automatic and all is good.

https://nuitka.net/doc/factory.html
